<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionMirroringForm</class>
 <widget class="QWidget" name="OptionMirroringForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="AddableListWidget" name="widget_NoTrn" native="true"/>
     </item>
     <item>
      <widget class="AddableListWidget" name="widget_NoDel" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox_FnameCnv">
       <property name="text">
        <string>ファイル名を全て小文字にして転送(&amp;L)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_UpDelNotify">
       <property name="text">
        <string>ミラーリングアップロードでファイル削除前に確認(&amp;U)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_DownDelNotify">
       <property name="text">
        <string>ミラーリングダウンロードでファイル削除前に確認(&amp;D)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_NoTransferContents">
       <property name="text">
        <string>ミラーリングでファイル内容を転送しない(&amp;F)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AddableListWidget</class>
   <extends>QWidget</extends>
   <header>ui/common/addablelistwidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
